payments_service/urls.py

```diff
@@ -1,15 +1,19 @@
 from django.conf.urls import include, url
 
 
-urlpatterns = [
+api_patterns = [
     url(r'^auth/',
         include('payments_service.auth.urls', namespace='auth')),
     url(r'^braintree/',
         include('payments_service.braintree.urls', namespace='braintree')),
-    url(r'',
+    url(r'^$',
```

Comment:
if you do want to leave it as ``r'^$'`` I think this should not use an include but instead should import the status index view directly.